src/Update.hs

```diff
@@ -248,11 +258,11 @@ publishPackage log updateEnv oldSrcUrl newSrcUrl attrPath result opDiff = do
   let metaDescription =
         if d == T.empty
           then ""
-          else "\n\nmeta.description for " <> attrPath <> " is: '" <> d <> "'."
+          else "\n\nmeta.description for " <> attrPath <> " is: " <> d
   let metaHomepage =
         if u == T.empty
           then ""
-          else "\n\nmeta.homepage for " <> attrPath <> " is: '" <> u
+          else "\n\nmeta.homepage for " <> attrPath <> " is: " <> u
```

Comment:
The values that come out of `nix eval` are already themselves quoted, so we don't need to add additional quotes.  If you squint you can see there are in fact extra quotes:
https://github.com/NixOS/nixpkgs/pull/80600